x/cdp/keeper/fees.go

```diff
@@ -27,6 +28,46 @@ func (k Keeper) CalculateFees(ctx sdk.Context, principal sdk.Coins, periods sdk.
 	return newFees
 }
 
+// TODO UpdateFeesForRiskyCdps
+// Select cdps with 10% of the liquidation ratio (ie. 150% * 0.1 = 15% = 165% or lower)
+// call calculate fees for each of those cdps
+// store the result of calculate fees in cdp.AccumulatedFees
+// set cdp.FeesLastUpdated to the current block time (ie. ctx.BlockTime())
+
+// UpdateFeesForRiskyCdps calculates fees for risky CDPs
+// The overall logic is first select the CDPs with 10% of the liquidation ratio
+// Then we call calculate fees on each of those CDPs
+// Next we store the result of the fees in the cdp.AccumulatedFees field
+// Finally we set the cdp.FeesUpdated time to the current block time (ctx.BlockTime()) since that
+// is when we made the update
+// TODO - QUESTION - this method signature should only take (ctx sdk.Context, cp types.Params) as parameters? Is this correct?
+// TODO - QUESTION - is types.CollateralParam the correct type?
+func (k Keeper) UpdateFeesForRiskyCdps(ctx sdk.Context, cp types.CollateralParam) {
+
+	// first calculate the target ratio based on liquidation ratio plus ten percent
+	value, err := sdk.NewDecFromStr("1.1")
+	if err != nil {
+		fmt.Printf("got error: %s", err) //  TODO - QUESTION - is there another method we should use for error logging instead of
```

Comment:
@karzak I just printed the error out here - is there an error logging function or anything that I should call instead?